src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/QualificationsAndTrainingResourceMapper.kt

```diff
@@ -94,17 +94,39 @@ abstract class QualificationsAndTrainingResourceMapper {
 
   fun toOffsetDateTime(instant: Instant?): OffsetDateTime? = instant?.atOffset(ZoneOffset.UTC)
 
+  fun toUpdatePreviousQualificationsDto(
```

Comment:
Is this the only mapper class that needed to change? Qualifications and Training mapper, which maps Previous Qualifications and Previous Training ?

I think from the PR description I was expecting to see changes like this in all of the "update" mappers. Presumably they already have this type of code and dont need to be changed?

Reply:
That's right - the existing mappers already cater for null values (except for `WorkOnReleaseResourceMapper` but that's because `workOnRelease` is not an object in the CIAG API - only in our domain).

Reply:
Cool, in that case, this PR is a work of art 😁